e2e/components/block-scroll-strategy/block-scroll-strategy.e2e.ts

```diff
@@ -1,13 +1,13 @@
 import {browser, Key, element, by} from 'protractor';
 import {screenshot} from '../../screenshot';
-import {getScrollPosition} from '../../util/query';
+import {getScrollPosition, asyncSpec} from '../../util/index';
 
 
 describe('scroll blocking', () => {
   beforeEach(() => browser.get('/block-scroll-strategy'));
   afterEach(() => clickOn('disable'));
 
-  it('should not be able to scroll programmatically along the x axis', async (done) => {
+  it('should not be able to scroll programmatically along the x axis', asyncSpec(async () => {
```

Comment:
According to @vikerman this `asyncSpec` wrapper should be unnecessary w/ jasminewd2